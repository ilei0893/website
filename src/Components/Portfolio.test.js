import React from "react";
import Portfolio from "./Portfolio.js";
import renderer from "react-test-renderer";

describe("Tests the portfolio section", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Portfolio />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
