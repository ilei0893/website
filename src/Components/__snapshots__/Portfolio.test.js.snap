// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the portfolio section matches the snapshot 1`] = `
<section
  id="portfolio"
>
  <div
    className="row"
  >
    <div
      className="twelve columns collapsed"
    >
      <h1>
        Check Out Some of My Works.
      </h1>
      <div
        className="bgrid-quarters s-bgrid-thirds cf"
        id="portfolio-wrapper"
      />
    </div>
  </div>
</section>
`;
